### PR TITLE
add chain id

### PR DESCRIPTION
**Pull request by fanyuexiao**
Closes #<issue number>.

### Reviewers

Main reviewers:

- @John
- @Doe

Codeowner reviewers:

- @-Alice
- @-Bob

### CODE REVIEW

File: server/service.go [hunk 1]
```diff
@@ -412,23 +439,26 @@ func (s *bridgeService) GetPendingTransactions(ctx context.Context, req *pb.GetP
 			BridgeToken:  deposit.OriginalAddress.Hex(),
 			TokenAmount:  deposit.Amount.String(),
 			EstimateTime: defaultTxEstimateTime,
-			Time:         uint64(deposit.Time.Unix()),
+			Time:         uint64(deposit.Time.Unix()) * 1000,
```
Comment (trunghai95) [inline server/service.go]: Can use `deposit.Time.UnixMilli()` instead

File: server/service.go [hunk 2]
```diff
@@ -370,38 +380,55 @@ func (s *bridgeService) GetCoinPrice(ctx context.Context, req *pb.GetCoinPriceRe
...
-		return nil, err
+		return &pb.CommonTransactionsResponse{
+			Code: defaultErrorCode,
+			Data: nil,
+		}, nil
+	}
+
+	hasNext := len(deposits) > int(limit)
+	if hasNext {
+		deposits = deposits[0:limit]
```
Comment (trunghai95) [inline server/service.go]: `deposits[:limit]` is enough (but your way is correct too)

File: proto/src/proto/bridge/v1/query.proto [hunk 3]
```diff
@@ -292,4 +294,36 @@ message GetPendingTransactionsResponse {
...
+    string error_code = 3;
+    string error_message = 4;
+    string detailMsg = 5;
+    repeated CoinInfo data = 6;
+}
+
+message CommonCoinPricesResponse {
+    uint32 code = 1;
+    string msg = 2;
+    string error_code = 3;
```
Comment (trunghai95) [inline proto/src/proto/bridge/v1/query.proto:320]: Why do we mix `snake_case` and `camelCase`? Should we unify all our fields to use the same case convention?
(Protobuf convention is normally `snake_case` btw)

**Comment by fanyuexiao** [comment]
1、add cors request headers
2、block time fix
3、add from/toChainId
4、add hasNext
5、use commonResponse struct

**Pull request merged**